packages/sdk-v2/src/sdk/goodid/react.ts

```diff
@@ -147,3 +145,80 @@ export const useAggregatedCertificates = (account: string): AggregatedCertificat
     });
   }, [certificates]);
 };
+
+// could be the configuration has to be set on the native side (on root-level)
+// at least the permission android.permission.ACCESS_FINE_LOCATION has to be added
+// in order for below hook to work
+
+// GeoLocation.setRNConfiguration({
+//   skipPermissionRequests: false,
+//   authorizationLevel: "whenInUse",
+//   locationProvider: "auto"
+// });
+
+export interface LocationState {
+  location: [number, number] | null;
+  error?: string | null;
+}
+
+export const useLocation = () => {
+  const [locationState, setLocationState] = useState<LocationState>({ location: null, error: null });
+
+  const onError = error => {
+    setLocationState({ location: null, error });
+  };
+
+  const getCurrentPosition = () => {
+    // based on the platform getCurrentPosition will use the appropriate API
+    GeoLocation.getCurrentPosition(
+      (position: any) => {
+        const { latitude, longitude } = position.coords;
+        setLocationState({ location: [latitude, longitude] });
+      },
+      (error: any) => {
+        onError(error);
+      },
+      {} // is optional but not made optional in the type definition
+    );
+  };
+
+  useEffect(() => {
+    console.log("platform", Platform.OS);
+    if (Platform.OS === "web") {
+      getCurrentPosition();
+      return;
+    }
+    // requestAuthorization only is needed on native
+    GeoLocation.requestAuthorization(getCurrentPosition, onError);
+  }, []);
+
+  return { locationState };
+};
+
+export const useGetCertificates = (account: string | undefined, baseEnv: any) => {
+  const { storeCertificate } = useCertificates(account ?? "");
+
+  const fetchCertificates = useCallback(
```

Comment:
request or issueCertificates, not fetch

Reply:
done here: https://github.com/GoodDollar/GoodWeb3-Mono/pull/136/commits/147356f77fe0049b9a7e3ef71c0b1280933089ca

Reply:
just return issueCertificates from the hook

Reply:
done here: https://github.com/GoodDollar/GoodWeb3-Mono/pull/136/commits/ecd2d963ac508b2d1411734189de18a0377ee5e2